src/site/content/en/blog/developer-satisfaction/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Web Developer Satisfaction
+subhead: 
+description: 
+date: 2021-03-22
+tags:
+  - blog
+draft: true
+---
+
+Web Developer Satisfaction, shortened DevSAT, is a Google project to gather
+information about the needs of web developers. Our goal is to make sure
+developers are being heard, know what's happening with their feedback and see
+issues being addressed. DevSAT is how we ensure we are focusing on the right
+areas with our Engineering and Developer Relations work. The goal is a more
+reliable, predictable and interoperable web platform, enabling developers to
+invest in and trust it, and adopt and use new features to grow the platform and
+their business.
+
+## Ongoing research to understand developer needs
+
++   [MDN DNA (Developer Needs Assessment) surveys](https://insights.developer.mozilla.org/)
++   [MDN Browser Compatibility Report 2020](https://insights.developer.mozilla.org/reports/mdn-browser-compatibility-report-2020.html)
++   MDN Web Testing Research is in progress, results will be announced soon
+
+## 2021 focus on compatibility
+
+Compatibility on the web has always been a big challenge for developers. The
+goal in 2021 is to eliminate browser compatibility problems in 5 key focus areas
+so developers can confidently rely on these areas not having bugs. This effort
+is called #Compat2021.
+
+You can follow along with the process in the Compat 2021 dashboard to see how
+each major browser engine are doing in the 5 areas:
+
+<iframe src="https://wpt.fyi/compat2021?embedded" 
```

Comment:
@robdodson I'm trying to make this iframe responsive, but I'm getting a horizontal scrollbar inside the iframe on some device widths (Iphone X for example).  Do we have some CSS classes I can slap on to make it work? [Glitch iframe wrapper](https://github.com/GoogleChrome/web.dev/search?l=JavaScript&q=glitch-embed-wrap&type=code) doesn't seem to do much beyond what I've done. Should I try `aspect-ratio`?

Reply:
@mihajlija were you able to get this to work with help from Adam and Una or is it still stuck?

Reply:
Apparently it can't be helped, the compat website has responsiveness issues. We've decided it's good enough as is for now.

Reply:
I've filed https://github.com/web-platform-tests/wpt.fyi/issues/2494 about it, maybe it can be fixed later.